<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgOpenAsciiFile</class>
 <widget class="QDialog" name="DlgOpenAsciiFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open ASCII File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblFileName">
         <property name="text">
          <string>FileName:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtFileName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFileTio">
       <property name="text">
        <string>Here are the first lines of this file. Choose each column attribution (one cloud at a time):</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="tblAsciiFile">
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sparator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtSeparator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAsciiCode">
         <property name="text">
          <string>(ASCII code %1)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolBtnSpace">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnTab">
           <property name="text">
            <string>Tab</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnComma">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolBtnSemicolon">
           <property name="text">
            <string>;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Skip Lines:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSkipLines"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Max number of points per cloud</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dblSpinBoxMaxPoints">
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string> Million</string>
         </property>
         <property name="minimum">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sampling Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbxSamplingRate">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/10</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushBtnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushBtnOk">
         <property name="text">
          <string>Ok</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnOk</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBtnSpace</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>ClickedSeparator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBtnTab</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>ClickedSeparator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBtnComma</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>ClickedSeparator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBtnSemicolon</sender>
   <signal>clicked()</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>ClickedSeparator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSeparator</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>SetSeparator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSkipLines</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>SetSkippedLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dblSpinBoxMaxPoints</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DlgOpenAsciiFile</receiver>
   <slot>SetMaxCloudSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateTable(const QString)</slot>
  <slot>ClickedSeparator()</slot>
  <slot>SetSkippedLines()</slot>
  <slot>SetMaxCloudSize()</slot>
  <slot>SetSeparator()</slot>
 </slots>
</ui>
